_data/nvram.yaml

```diff
@@ -1,12 +1,29 @@
 ---
 - name: AAPL,ignore
+- platforms:
+  - macOS T2
```

Comment:
My suggestion here is to decompose...

For example in this case I'd mark it as

- macOS
- Intel
- bridgeOS

You would then select T2 specific variables for the T2 itself with "bridgeOS AND !macOS" and the macOS EFI variables as "macOS AND bridgeOS"